examples/cifar10/README.md

```diff
@@ -1,142 +1,52 @@
-  -  `pyramid`: PyramidNet, pyramid alpha=200, 272 layers, ShakeDrop
-        regularization
-- `--wrn_dropout_rate` (default=0.3): DropOut rate used in Wide ResNet
-    (on all residual blocks)
-- `--rng` (default=0): Random seed used for network initialization and
-    stochasticity during training
-
-
-## Issues
-
-We apply L2 regularization to weights and biases, whereas it is mostly *not*
-applied to biases.
-
-
-## References
-
-Various code was used as a reference to help replicate baseline results.
-In particular the Fast AutoAugment codebase was particularly helpful
-as its config files describe training regimes for the wide ResNet
-with shake-shake and PyramidNet baselines.
-
-Shake-shake author's implementation:
-https://github.com/xgastaldi/shake-shake
-
-PyTorch implementation of shake-shake:
-https://github.com/owruby/shake-shake_pytorch
-
-Fast AutoAugment that uses a number of different models:
-https://github.com/kakaobrain/fast-autoaugment
+## CIFAR10 classification
+Trains a **family** ResNet-style models (He *et al.*, 2015; Zagoruyko and Komodakis, 2017; Han *et al.*, 2017) for the CIFAR10 classification task (Krizhevsky, 2009).
+
+This example implements several architectures, regularization methods (Gastaldi, 2017; Yamada *et al.*, 2018) and learning rate schedules that can be used in various combinations.
+
+### Requirements
+* TensorFlow dataset `cifar10`
+
+### Supported setups
+The model should run with other configurations and hardware, but explicitly tested on the following.
+
+#### Wide ResNet: 26 layers, 10x width (Zagoruyko and Komodakis, 2017)
+| Hardware | Epochs | Learning rate | Training time | Error rate | TensorBoard.dev |
+| --- | --- | --- | --- | --- | --- |
+| 1 x Nvidia V100 (16GB) | 200 | Piece-wise constant | 4h 36m | 4.45% | [2020-03-22](https://tensorboard.dev/experiment/1kE2bq9RR7WrT7Zw14fNpg/) |
+| 8 x Nvidia V100 (16GB) | 200 | Piece-wise constant | 57m | 3.93% | [2020-03-22](https://tensorboard.dev/experiment/IpzU0txnR7uZnExGmk7IEA/) |
+
+#### Wide ResNet: 26 layers, 6x width, Shake-Shake regularization (Gastaldi, 2017)
+| Hardware | Epochs | Learning rate | Training time | Error rate | TensorBoard.dev |
+| 1 x Nvidia V100 (16GB) | 200 | Piece-wise constant | 3h 38m | 3.43% | [2020-03-22](https://tensorboard.dev/experiment/dJc9e4k1R5mC0DPVZyYrpg/) |
+| 8 x Nvidia V100 (16GB) | 200 | Piece-wise constant | 54m | 3.39% | [2020-03-26](https://tensorboard.dev/experiment/l2CPAqpnTlCjjZKbnuOomg/) |
+| 1 x Nvidia V100 (16GB) | 1800 | Cosine | 1d 9h 25m | 2.97% | [2020-03-22](https://tensorboard.dev/experiment/aXyhYX2oSxKH5lIlj4Ao5g/) |
+| 8 x Nvidia V100 (16GB) | 1800 | Cosine | 8h 5m | 2.82% | [2020-03-26](https://tensorboard.dev/experiment/kq303ri1RHygWxb5YrLp1Q/) |
+
+#### PyramidNet, Shake-drop regularization (Han *et al.*, 2017; Yamada *et al.*, 2018)
+| 8 x Nvidia V100 (16GB) | 300 | Piece-wise constant | 6h 41m | 3.25% | [2020-03-24](https://tensorboard.dev/experiment/OpZLDnVjRhmJKXq62RoaUQ/) |
```

Comment:
These don't render as tables for some reason (also they don't have the row with --- | ---, maybe that's related?)

Reply:
Fixed. Thanks for spotting this!